tests/utils/device.test.js

```diff
@@ -36,3 +36,21 @@ suite('isIOSOlderThan10', function () {
     assert.notOk(device.isIOSOlderThan10(v12));
   });
 });
+
+suite('environment', function () {
+  test('isNodeEnvironment is false for browser tests', function () {
+    assert.strictEqual(device.isNodeEnvironment, false);
```

Comment:
I would like to test the correct type as well. I don't want to accidently return a truthy/falsy object (which I almost did: `process && process.browser` was returning `undefined` instead of `false`).